File: Arrays/1877_minimize_mamaximum_pair_sum_in-array.js
var minPairSum = function(nums) {
    nums.sort((a,b)=>a-b)
    let highest=[];
    for(let i=0,j=nums.length-1;i<j&&j>i;i++,j--){
        highest.push(nums[i]+nums[j])
    }
    return Math.max(...highest)
};

console.log(minPairSum([3,5,4,2,4,6]))

File: Arrays/1877_minimize_mamaximum_pair_sum_in-array.py
class Solution(object):
    def minPairSum(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        nums.sort()
        highest = []

        for i in range(len(nums) // 2):
            highest.append(nums[i] + nums[len(nums) - 1 - i])

        return max(highest)
        